ethereum/eth/src/test/java/org/hyperledger/besu/ethereum/eth/manager/EthProtocolManagerTest.java

```diff
@@ -272,6 +279,40 @@ public void disconnectNewPoWPeers() {
     }
   }
 
+  @Test
+  public void disconnectOnMissingBlockRangeWhenEth69() {
```

Comment:
what happens if capability is eth/69 but the peer still sends a status message with total difficulty? is that case worth a test?